.github/workflows/icfy-stats.yml

```diff
@@ -32,7 +32,6 @@ jobs:
           NODE_ENV: production
           BROWSERSLIST_ENV: defaults
           WORKERS: 2
-          WEBPACK_OPTIONS: --progress=profile
```

Comment:
I'm curious what these `WEBPACK_OPTIONS` did and how removing them is related to Yarn v3

Reply:
`WEBPACK_OPTIONS` was used when running webpack:

```
"build-client-do": "webpack --config client/webpack.config.js --stats-preset errors-only $WEBPACK_OPTIONS",
```

I got an `Unbound variable "WEBPACK_OPTIONS"` error in yarn v3 when the env var is not defined. I tried to provide a default value using bash semantics (`"build-client-do": "webpack ... ${WEBPACK_OPTIONS:-}"`) but I couldn't get it to work.

I assumed we were not really using the profiling log in ICFY (after all, we have the profiling enabled in the main TeamCity build and I don't see why we want to get the same info twice), so I took the path of least resistance and removed it. If we want to bring it back, we can instead set `PROFILE: true` and it should have the same effect.